pkgs/top-level/python-packages.nix

```diff
@@ -17550,6 +17550,10 @@ self: super: with self; {
     inherit (pkgs) graphviz;
   };
 
+  xen = toPythonModule (pkgs.xen-slim.override {
```

Comment:
Using xen-slim causes duplication of xen package in system closure, but xen can't be built with internal qemu on latest python.

I haven't found a good way to only build python package out of xen source tree, but it might be made better in future with dedicated python output.

Reply:
I'm still not exactly sure why we're shipping 24.11 with a clearly broken Meson. Have you tried using 3.12 but overriding the python package to include `setuptools`? I tried it myself a few weeks ago but didn't have any success, but it's probably because I'm not very familiar with the python ecosystem.

Reply:
I have tried adding setuptools to meson package itself, and it didn't work for me. Meson does something clever/stupid and can't find setuptools this way.

Reply:
...Python module is not required for qubes, as qubes has its own qubes-vmm-xen based off xen-slim, and we can only include relevant xen changes, but I wanted to add xen python package itself for easier testing.